src/GLFWDotNet/GLFW.LoadAssembly.cs

```diff
@@ -16,6 +15,72 @@ private static class Win32
             public static extern IntPtr GetProcAddress(IntPtr module, string procName);
         }
 
+        private static class Linux
+        {
+
+            const int RTLD_NOW = 2;
+            public static IntPtr LoadLibrary (string fileName)
+            {
```

Comment:
Please remove the space between the method name and the open paren ( for consistency.